src/app/components/LayerDetailsModal/index.tsx

```diff
@@ -0,0 +1,182 @@
+import { FC } from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { RouteUtils } from '../../utils/route-utils'
+import { COLORS } from '../../../styles/theme/colors'
+import { Circle } from '../Circle'
+import CheckIcon from '@mui/icons-material/Check'
+import { useTranslation } from 'react-i18next'
+import { styled, useTheme } from '@mui/material/styles'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import HighlightOffIcon from '@mui/icons-material/HighlightOff'
+import IconButton from '@mui/material/IconButton'
+import Button from '@mui/material/Button'
+import { useLayerParam } from '../../hooks/useLayerParam'
+import { BlockchainIcon } from '../BlockchainIcon'
+import Link from '@mui/material/Link'
+import { Link as RouterLink } from 'react-router-dom'
+import { Layer } from 'oasis-indexer/api'
+import { TFunction } from 'i18next'
+import { Modal } from '../Modal'
+
+const CloseModalIconButton = styled(IconButton)(({ theme }) => ({
+  position: 'absolute',
+  top: theme.spacing(0),
+  right: theme.spacing(0),
+  [theme.breakpoints.up('sm')]: {
+    top: theme.spacing(5),
+    right: theme.spacing(5),
+  },
+}))
+
+const EnabledLayersGrid = styled(Box)(({ theme }) => ({
+  display: 'grid',
+  gridTemplateColumns: 'fit-content(70%) 1fr',
+  '&:not(:last-child)': {
+    marginBottom: theme.spacing(5),
+  },
```

Comment:
This seems to be a grid for only one layer. I think it would make more sense to make it for all enabled layers. And use row-gap instead?